styles/scss/components/application/widgets/taskEditor.module.scss

```diff
@@ -112,11 +134,17 @@
 }
 
 .scheduler_quickbutton {
-    padding: .5rem .9rem;
-    display: flex;
-    justify-content: space-between;
-    font-size: 1.3rem;
-    cursor: pointer;
+  line-height: normal;
+  padding: 0.5rem 0.5rem;
```

Comment:
Please use 
![CleanShot 2024-02-23 at 09 14 19@2x](https://github.com/zli213/TaskTick/assets/127060502/84f8c65b-7d11-4bfe-bb59-46c1999efa7f)
here

Reply:
This file contains many similar issues; I suggest making a uniform update.